packages/react-server-cli/src/handleCompilationErrors.js

```diff
@@ -0,0 +1,28 @@
+import reactServer from "./react-server";
+
+const logger = reactServer.logging.getLogger(__LOGGER__);
+
+// takes in the err and stats object returned by a webpack compilation and returns
+// an error object if something serious happened, or null if things are ok.
+export default function handleCompilationErrors (err, stats){
+	if (err) {
+		logger.error("Error during webpack build.");
+		logger.error(err);
+		return new Error(err);
```

Comment:
Meh.  The only reason for the `ConfigurationError` is to provide a clean UI when we short circuit out due to a simple problem (file already exists, etc).  For ordinary errors where stack trace is actually desirable I don't see any need to subclass.